src/js/_enqueues/lib/emoji-loader.js

```diff
@@ -115,23 +115,26 @@
 				return ! isIdentical;
 			case 'emoji':
 				/*
-				 * Burning Love: Just a hunk, a hunk of burnin' love.
+				 * Why can't we be friends? Everyone can now shake hands in emoji, regardless of skin tone!
 				 *
-				 *  To test for Emoji 13.1 support, try to render a new emoji: Heart on Fire!
+				 * To test for Emoji 14.0 support, try to render a new emoji: Handshake: Light Skin Tone, Dark Skin Tone.
 				 *
-				 * The Heart on Fire emoji is a ZWJ sequence combining ❤️ Red Heart, a Zero Width Joiner and 🔥 Fire.
+				 * The Handshake: Light Skin Tone, Dark Skin Tone emoji is a ZWJ sequence combining 🫱 Rightwards Hand,
+				 * 🏻 Light Skin Tone, a Zero Width Joiner, 🫲 Leftwards Hand, and 🏿 Dark Skin Tone.
 				 *
-				 * 0x2764, 0xfe0f == Red Heart emoji.
-				 * 0x200D == Zero-Width Joiner (ZWJ) that links the two code points for the new emoji or
+				 * 0x1FAF1 == Rightwards Hand
```

Comment:
Using the rightward hand emoji may not work for these tests as it was introduced in Emoji 14.

> When updating this test for future Emoji releases, ensure that individual emoji that make up the sequence come from older emoji standards.

I guess the reason is that some systems may end up comparing `[broken emoji]` to `[broken emoji]` and returning a false positive.

As I didn't write the comment, guess is the key word though.

Reply:
I was concerned about that initially (and why I didn't put anything in there originally; there are no new emoji that are composite of only older emoji), but it did accurately catch for me.

Once we get the assets on the CDN, we can use Browserstack or something similar to test a wider range of older platforms as confirmation.

Reply:
Let's land and test during the beta.